cypress/e2e/dashboard/chapters/[chapterId]/edit.cy.ts

```diff
@@ -73,4 +73,28 @@ describe('chapter edit dashboard', () => {
       cy.contains(chapterData.name);
     });
   });
+
+  it('deny admins to delete a chapter', () => {
+    const chapterId = 1;
+    cy.login(Cypress.env('JWT_ADMIN_USER'));
```

Comment:
```suggestion
    cy.login('admin@of.chapter.one);
```
The login function is a little more reasonable these days, you just pass it the email address you want to log in as.